guides/common/modules/proc_deploying-a-virt-who-configuration-on-a-separate-rhel-server.adoc

```diff
@@ -0,0 +1,53 @@
+[id="deploying-a-virt-who-configuration-on-a-separate-rhel-server"]
+= Deploying a virt-who configuration on a dedicated {EL} server
+
+For VMware vSphere, Microsoft Hyper-V, and Nutanix AHV, you can deploy the virt-who configuration on a dedicated {EL} server.
+The server can be physical or virtual.
+
+Global configuration values apply to all virt-who configurations on this server and are overwritten if you deploy a new virt-who configuration.
+
+.Prerequisites
+* You have registered the {EL} server to {ProjectServer}. For more information, see {ProvisioningDocURL}registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts[Registering hosts by using global registration] in _{ProvisioningDocTitle}_.
+
+.Procedure
+. Add a TCP port for communication between virt-who and {ProjectServer} to the runtime configuration:
++
+[options="nowrap"]
+----
+$ firewall-cmd --add-port="<port>/tcp" <1>
+----
+<1> Specify `443` for VMware vCenter and Nutanix AHV. Specify `5985` for Microsoft Hyper-V.
+. Add a TCP port for communication between virt-who and {ProjectServer} to the permanent configuration:
++
+[options="nowrap"]
+----
+$ firewall-cmd --add-port="<port>/tcp" --permanent
```

Comment:
Is this executed on the `{ProjectServer}` or on the `{EL} server`? My guess would be the latter, but I think this would benefit from being more explicit.

Reply:
I'm looking at https://github.com/candlepin/virt-who/blob/4e3360c5db9b567f566d681fc8786531af12511d/virt-who.8#L116-L132 and my reading form that this is on the target hypervisor and not `virt-who` -> `{ProjectServer}`.

So for VMware you must ensure the host running virt-who can access VMware on port 443. AFAIK you can't run `firewall-cmd` on VMware itself. You do need to allow outbound traffic, but in most network configs that's allowed by default.

My recommendation would be to drop `firewall-cmd` and instead say that `server.example.com` must have access to the hypervisor on a certain port, then give an overview of which ports.